### PR TITLE
Bug/tool not working if age is missing

### LINKED ISSUE #317: [BUG] Tool not working if age variable is not included in the data
**Describe the Bug**

It seems that the current version of the tool crashes if age variable is not included in the data. However, since age is not a mandatory variable this should not happen. 

**To Reproduce**

1. Upload data without age variable
2. Observe the errors in the Input parameters tab
3. Also, in the console of the RStudio you should see a message: "Warning: Error in dplyr::mutate: i In argument: `dplyr::across(...)`.
Caused by error in `dplyr::all_of()`:
! Can't subset columns that don't exist.
x Column `age` doesn't exist."

**Expected Behavior**

App does not work as it should.

### PR DESCRIPTION
**Pull request by aj-thl**
Fixed a bug so that the tool works again when the age variable is not included in the data.

Fixes #317

### CODE REVIEW

File: NEWS.md [hunk 1]
```diff
@@ -7,6 +7,7 @@
 * Fixing bug so that plot_time_series() works with upperbond_pad and expected_pad being all NA
 * Added notification about negative values in age.
 * Improvements in the documentation of `build_signals_table()`, `prepare_signals_agg_table()`
+* Bug fix inside `preprocess_data()` so that the tool works again when the age variable is not included in the data.
```
Comment (tinneuro) [inline NEWS.md:10]: Can you put this on top of SignalDetectionTool 0.3.0 so that it is clear that it belongs to the next version?

**Comment by tinneuro** [comment]
Great thanks for your contribution!

File: NEWS.md (hunk 1 above)
Comment (tinneuro) [reply NEWS.md:10]: and just pull develop and merge in your branch to resolve conflicts as there was one update of develop in the meantime ;)

Comment (aj-thl) [reply NEWS.md:10]: Sure! I'll let you know when these are done.

**Comment by aj-thl** [comment]
Ok, so I updated the NEWS and merged develop :)

**Review by tinneuro — APPROVED** [review]

**Pull request merged**